.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,11 @@
+name: Auto approve
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@v2.0.0
+      if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
```

Comment:
Should we limit this to non-breaking changes? Breaking changes are likely worth a sanity check. Although if it's just dev requirements, maybe it doesn't matter?

Reply:
Those options can be set in the dependabot config yaml or settings https://app.dependabot.com/accounts/tophat/projects/164315